src/App.tsx

```diff
@@ -17,18 +50,21 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                status={status}
+                onStatusChange={setStatus}
+                query={query}
+                onQuerryChange={setQuery}
```

Comment:
There is a typo in the prop name `onQuerryChange`. It should be `onQueryChange` to match the naming convention and avoid potential bugs. Please update the prop name in both the `App` component and the `TodoFilter` component to ensure consistency.